gui/CameraView.cxx

```diff
@@ -607,6 +607,13 @@ void CameraView::addFeatureTrack(kwiver::vital::track const& track)
   d->updateFeatures(this);
 }
 
+void CameraView::setFrameName(QString frameName)
```

Comment:
Missing separator line... @ALouis38, can you please pay attention to maintaining style consistency with the files you are editing? Thanks!